src/tracing-policy.ts

```diff
@@ -14,65 +14,80 @@
  * limitations under the License.
  */
 
-'use strict';
-
-function RateLimiterPolicy(samplesPerSecond) {
-  if (samplesPerSecond > 1000) {
-    samplesPerSecond = 1000;
-  }
-  this.traceWindow = 1000 / samplesPerSecond;
-  this.nextTraceStart = Date.now();
+/**
+ * An object that determines whether a request should be traced.
+ */
+export interface Policy {
```

Comment:
Could we use a more descriptive name here?

Reply:
Done